types/fundamental/real32trunc/README.md

```diff
@@ -0,0 +1,21 @@
+# Real Column Types with variable width and truncated mantissa
+
+## Fields
+
+ * `FloatReal32Trunc10`
+ * `FloatReal32Trunc16`
+ * `FloatReal32Trunc31`
+ * `DoubleReal32Trunc10`
+ * `DoubleReal32Trunc16`
+ * `DoubleReal32Trunc31`
```

Comment:
Do we have (do we support any) `DoubleReal64Trunc43`  ?

Reply:
No, the "hierarchy" is `Real64` > `Real32` > `Real32Trunc`. I guess we didn't see a potential use case to keep 11 bits of exponent or more than 23 bits of mantissa...

Reply:
Fair enough (Although I suppose someone could want to have high dynamic range with few significant digt for some reason i.e. small mantissa but full exponent use) (Until someone actually this is of course moot :) )